docs/releases/network-upgrades.md

```diff
@@ -8,100 +8,104 @@ For details on why and how network upgrades happen, along with explanations of t
 
 ## Upcoming upgrades
 
-### Increased work difficulty
+No upcoming upgrades at this time
 
-To help ensure Quality of Service on the network by increasing the difficulty required for send and change blocks to be considered valid by the network (8x compared to current). To help offset the difficulty increase and add incentive to receive blocks so ledger pruning can be done more broadly in the future, the difficulty for receive blocks will simultaneously be reduced (1/8 compared to current).
+---
+
+## Future upgrades
 
-#### Transition details
+### New PoW algorithm
 
-This upgrade is sometimes referenced as the epoch v2 upgrade and the relate events to complete are as follows:
+**Purpose**
+
+To help ensure Quality of Service on the network by providing the ability to generate and validate PoW using a new algorithm. This algorithm will be more memory-hard causing the resources required for generating transactions to shift accordingly.
+
+**Transition details**
 
 | Date | Type | Description |
 |------|------|-------------|
-| <span class="no-break">2020-06-16</span>  | Node release | Nano node V21.0 released which includes changes necessary for supporting new difficulty validation and generation |
-| <span class="no-break">2020-08-18</span> | <span class="no-break">v2 epoch blocks distribution start</span> | Distribution of v2 epoch blocks to all accounts to mark in the ledger the point at which the new work difficulty levels will be required. The start of this distribution process will occur once key services and over 90% of voting weight on the network has upraded. |
-| <span class="no-break">2020-08-29</span> | <span class="no-break">v2 epoch blocks distribution end</span> | Distribution of epoch blocks ends after all accounts are upgraded. |
+| TBD  | Node release | Nano node V20.0 released to include a new PoW server and epoch block support. These are only foundational updates and will not be configured for activation in this release. |
```

Comment:
Slightly unrelated, but this TBD date can be updated and another line added with details since this epoch was used up by the difficulty update.

Reply:
Updated details there